data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,265 @@ event "remnant: pantica void sprites"
+			choice
+				`	"Anything unusual in the data?"`
+				`	"How much longer does the equipment need?"`
+					goto astralfinish
+			label astraldata
+			`	"There are many oddities about this place. Every visit helps revise our physics models a little closer. But they also reveal problems." She frowns, her gestures taking on a curious tone. "For instance, you will note that your ramscoops are not picking up anything. According to standard astronomical theory, this system should be a veritable river of suitable material, and yet there is nothing. Not even the normal background trace amounts we find in deep space. We have visited other black holes, and they conform to the standard models in this respect. But not Sagittarius A*." She pauses, thinking about it for a moment.`
+			choice
+				`	"So this is not the first expedition here?"`
+					goto astralfirst
+				`	"How much longer does the equipment need?"`
+			label astralfinish 
+			`	"The equipment will be finished shortly, Captain <first>. I just need to make a a few observations and we will be able to depart. In the meantime, I would recommend against maintaining an active cloak or engaging any systems that consume fuel. For whatever reason, Sagittarius A* continues to be the one place that the stellar wind is utterly devoid of anything to collect as fuel."`
+			`	"I will be going outside for a few minutes." Dawn declares. "I shall just be on the hull."`
+			choice
+				`	"Let me know when you are done."`
+					goto astraldone
+				`	"Would you mind if I came too?"`
+			`	"Not at all, Captain. The view should be remarkable, and so long as we stay inside the shields, the radiation will not pose a problem." You quickly don your spacesuit, and join Dawn outside on the hull. As promised, the view is indeed magnificent. Dawn signs ocassionally, pointing out various features, and you appreciate that out here, sound would just be incongruous with the natural splendor.`
+			label astraldone
+			`	After what feels like hours the equipment notifies you that it has finished taking its readings, and Dawn packs up the data in carefully shielded data crystals. Everything is ready to return to <destination>.`
+	on complete
+		payment 500000
+		conversation
+			`The journey back to <planet> was blur as the <ship> jumped through countless systems. Gradually the brilliant density of stars that make up the inner core faded into the ominous and yet comfortingly familiar red glow of the Ember Waste. Finally you slide into the approach patterns of <planet>, quickly tapping out your code into the IFF system and watching as the protective gaze of the anti-air emplacements sweep the skies around you for shadowing craft.`
```

Comment:
Good to see some acknowledgement of this brilliance.

Reply:
Thank you. I have been paying attention to all the descriptive talk that you and ravenshining and various others have been having about star density and the like. 🌟